### ISSUE TITLE
Guides - Migrate from Browserify

### ISSUE THREAD

**Issue by Scarysize** [webpack/webpack.js.org#41]
I think this would help a lot of people. Both tools try to accomplish similar things, though using very different concepts.

_User story_
As browserify user I want to easily be able to migrate over to webpack. I want to generate (transpiled) javascript bundles and set up a local development environment.

_Hints_
- Keep it simple
- How to generate a production bundle
- How to setup webpack-dev-server
- How to transpile ES6 code with babel

I would consider everything beyond these as "advanced".

**Comment by SpaceK33z** [comment]
Great idea.

> How to setup webpack-dev-server

I think the guide should link to the "How to develop?" page for this. It isn't done yet, but will be in not too long (#6). That page will explain in layman terms the ways to setup a local dev environment.

**Comment by bebraw** [comment]
Yeah, better to mention specific differences here and link to other pages where possible.

**Comment by skipjack** [comment]
@Scarysize would you be interested in submitting a pr to add this guide? I can facilitate the process...

**Comment by Scarysize** [comment]
@skipjack I will see if I can think of an concise guide. Though IMO the new documentation has improved tremendously, so it might not event be necessary anymore.

**Comment by skipjack** [comment]
> Though IMO the new documentation has improved tremendously, so it might not event be necessary anymore.

Happy to hear that you're finding the new documentation useful. I feel the same way in regards to this not being necessary but wanted to give you the option if you still felt strongly about it. I think if our core documentation is solid, migration guides like this shouldn't be needed. In general I think the focus should be core docs for now and, if people are still asking for third-party integration/migration docs, we can add those later on.

I'm going to close but if you do decide to write one and want to brainstorm ideas, feel free to re-open.

**Comment by Scarysize** [comment]
Sounds good!
